my-app/src/app/resume/page.tsx

```diff
@@ -0,0 +1,90 @@
+import Image from 'next/image'
+import styles from './page.module.css'
+
+export default function Resume() {

```

Comment:
This page looks really good! Something to consider is thinking about the duplicate code for each the different resume sections ( experience, skills, education etc)- could be something to put into a boxed component with props.